ipaserver/install/dnskeysyncinstance.py

```diff
@@ -164,6 +165,87 @@ def __setup_dnssec_containers(self):
 
         self._ldap_mod("dnssec.ldif", {'SUFFIX': self.suffix, })
 
+    def __are_named_options_configured(self, options):
+        """
+        Check whether sysconfig of named is patched or not.
+        Additional command line options for named are passed
+        via OPTIONS env variable. Since custom options can be
+        supplied by a vendor, at least, the base parsing of such
+        is required.
+        Current named command line options:
+        NS_MAIN_ARGS "46A:c:C:d:D:E:fFgi:lL:M:m:n:N:p:P:sS:t:T:U:u:vVx:X:"
+        If there are several same options the last passed wins.
+        """
+        if options:
+            pattern = r"[ ]*-[a-zA-Z46]*E[ ]*(.*?)(?: |$)"
+            p = re.compile(pattern)
+            engines = re.findall(p, options)
```

Comment:
I see,
Lib/re.py:
```python
231 def findall(pattern, string, flags=0):                                          
232     """Return a list of all non-overlapping matches in the string.              
233                                                                                 
234     If one or more capturing groups are present in the pattern, return          
235     a list of groups; this will be a list of tuples if the pattern              
236     has more than one group.                                                    
237                                                                                 
238     Empty matches are included in the result."""                                
239     return _compile(pattern, flags).findall(string)
```
Thank you.

Reply:
Fixed